docs/sources/docker-hub/official_repos.md

```diff
@@ -1,189 +1,110 @@
-image is ready for primetime, please contact us at
-[partners@docker.com](mailto:partners@docker.com) to have your project
-considered for the Official Repos program.
+# Official Repositories on Docker Hub
+
+The Docker [Official Repositories](http://registry.hub.docker.com/official) are
+a curated set of Docker repositories that are promoted on Docker Hub and
+supported by Docker, Inc. Docker's Official Repositories:
+
+* Provide essential base images (for example,
+  [`ubuntu`](https://registry.hub.docker.com/_/ubuntu/),
+  [`centos`](https://registry.hub.docker.com/_/centos/)) that serve as the
+  starting point for the majority of users. Many of the Official Repositories
+  are based on [`debian`](https://registry.hub.docker.com/_/debian/).
+
+* Provide drop-in solutions for popular programming language runtimes, data
+  stores, and other services, similar to what a Platform-as-a-Service (PAAS)
+  would offer.
+
+* Exemplify [`Dockerfile` best practices](/articles/dockerfile_best-practices)
+  and provide good documentation that serve as reference examples for other
+  image authors.
+
+* Ensure that security updates are applied in a timely manner. This is
+  particularly important as many Official Repositories are some of the most
+  popular on Docker Hub.
+
+* Provide a channel for software vendors to redistribute up-to-date and
+  supported versions of their products. Organization accounts on Docker Hub can
+  also serve this purpose, without the careful review or restrictions on what
+  can be published.
+
+Docker, Inc. sponsors a dedicated team that is responsible for reviewing and
+publishing all Official Repositories content. This team works in collaboration
+with upstream software maintainers, security experts, and the broader Docker
+community.
+
+While it is preferrable to have upstream software authors maintaining their
+corresponding Official Repositories, this is not a strict requirement. Creating
+and maintaining images for Official Repositories is a public process. It takes
+place openly on GitHub where participation is encouraged. Anyone can provide
+feedback, contribute code, suggest process changes, or even propose a new
+official repository.
+
+## Should I use Official Repositories?
+
+New Docker users are encouraged to use the Official Repositories in their
+projects. These repositories have clear documentation, promote best practices,
+and are designed for the most common use cases. Advanced users are encouraged to
+review the Official Repositories as part of their `Dockerfile` learning process.
+
+A common rationale for diverging from Official Repositories is is to optimize
+for image size. For instance, many of the programming language stack images
+contain a complete build toolchain to support installation of modules that
+depend on optimized code. An advanced user could build a custom image with just
+the necessary pre-compiled libraries to save space and reduce attack vectors in
```

Comment:
Nice catch.  I took out the security tidbit at the end to be more consistent/concise.